build/external-dependencies.props

```diff
@@ -117,17 +117,14 @@
     <CoreFxLabFeed>https://dotnet.myget.org/f/dotnet-corefxlab/api/v3/index.json</CoreFxLabFeed>
   </PropertyGroup>
   <ItemGroup>
-    <ExternalDependency Include="System.Buffers.Experimental" Version="$(SystemBuffersExperimentalPackageVersion)" Source="$(CoreFxLabFeed)" Private="true" Mirror="true" />
```

Comment:
You can also delete these variables from dependencies.props if they are not used anywhere else in this repo.

https://github.com/aspnet/Universe/blob/dd2cc91c10333f08cb42f8bd6466123816f48d22/build/dependencies.props#L121